### ISSUE TITLE
System.import and es6 template strings [webpack 2]

### ISSUE THREAD

**Issue by arshtepe** [webpack/webpack#2762]
**I'm submitting a bug report**
**I'm submitting a feature request**

**Webpack version:**
2.1.0-beta.15

**Please tell us about your environment:**
 Linux 

**Current behavior:**

``` js
System.import(`brace/theme/${themeName}`)
```

```
WARNING in xml-editor/XMLEditor.js
106:15 Critical dependency: the request of a dependency is an expression
```

**Expected/desired behavior:**
Convert to

``` js
System.import("brace/theme/"+ themeName)
```
- **Language:**ES6/7

**Comment by arthuralee** [comment]
Took a stab at this over at #3325. Not ready to merge yet - need more testing and need to update the lint/build/test pipeline to support ES6.

**Comment by Kovensky** [comment]
This is already in progress at #3297 (I probably should have linked here)

**Comment by TheLarkInn** [comment]
Looks like this is fixed with #3297
